k8s/logging/fluent-bit.yaml

```diff
@@ -0,0 +1,203 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluent-bit
+  namespace: logging
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::588562868276:role/fluent-bit-role
+---
```

Comment:
Minor style tweak: add a newline before each subsequent resource.  Makes it easier to pick out new resources as you scroll around.